### PR TITLE
chore: ux improvements

### PR DESCRIPTION
**Pull request by tyyzqmf**
----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

## Summary

1. display MSK refresh button
2. enforce at least 2 for the maximum for ingestion server
3. filter icon in Retention analysis
4. embed chart placehoder & increase create dashboard wait time to 10s
5. funnel default chart type is bar
6. delete path explore platform and fix node type data
7. auto receive value when input focus is removed from the input

## Implementation highlights

(describe how the merge request does for feature changes, share the RFC link if it has)

## Test checklist

- [ ] add new test cases
- [ ] all code changes are covered by unit tests
- [ ] end-to-end tests
  - [x] deploy web console with CloudFront + S3 + API gateway
  - [ ] deploy web console within VPC
  - [ ] deploy ingestion server
    - [ ] with MSK sink
    - [ ] with KDS sink
    - [ ] with S3 sink
  - [ ] deploy data processing
  - [ ] deploy data modeling
    - [ ] new Redshift Serverless
    - [ ] provisioned Redshift
    - [ ] Athena
  - [ ] deploy with reporting

## Is it a breaking change

- [ ] add parameters without default value in stack
- [ ] introduce new service permission in stack
- [ ] introduce new top level stack module

## Miscellaneous

- [ ] introduce new symbol link source file(s) to be shared among infra code, web console frontend, and web console backend

### CODE REVIEW

File: frontend/src/pages/pipelines/create/steps/buffer/BufferMSK.tsx [hunk 1]
```diff
@@ -212,7 +212,7 @@ const BufferMSK: React.FC<BufferMSKProps> = (props: BufferMSKProps) => {
                                 filteringType="auto"
                               />
                             </div>
-                            {update ? (
+                            {!update ? (
```
Comment (chenhaiyun) [inline frontend/src/pages/pipelines/create/steps/buffer/BufferMSK.tsx:215]: This is a history bug?

File: frontend/src/pages/analytics/comps/ExploreEmbedFrame.tsx [hunk 2]
```diff
@@ -73,7 +76,20 @@ const ExploreEmbedFrame: React.FC<ExploreEmbedFrameProps> = (
     }
   }, [embedUrl]);
 
-  return <div id={'EmbedId'} className="iframe-explore"></div>;
+  return (
+    <div id={'EmbedId'} className="iframe-explore">
```
Comment (chenhaiyun) [inline frontend/src/pages/analytics/comps/ExploreEmbedFrame.tsx:80]: id = 'EmbedId', no need {}

File: frontend/src/pages/analytics/comps/ExploreEmbedFrame.tsx [hunk 3]
```diff
@@ -10,8 +10,11 @@
  *  OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions
  *  and limitations under the License.
  */
+import { Box, SpaceBetween } from '@cloudscape-design/components';
 import { createEmbeddingContext } from 'amazon-quicksight-embedding-sdk';
+import ExtendIcon from 'components/common/ExtendIcon';
 import i18n from 'i18n';
+import { t } from 'i18next';
```
Comment (chenhaiyun) [inline frontend/src/pages/analytics/comps/ExploreEmbedFrame.tsx:17]:
```
import { useTranslation } from 'react-i18next';
const { t, i18n } = useTranslation();
```

File: frontend/src/pages/pipelines/create/steps/buffer/BufferMSK.tsx (hunk 1 above)
Comment (tyyzqmf) [reply frontend/src/pages/pipelines/create/steps/buffer/BufferMSK.tsx:215]: yes

File: src/control-plane/backend/lambda/api/service/reporting.ts [hunk 4]
```diff
@@ -127,6 +127,10 @@ export class ReportingService {
 
       const result = await this._buildFunnelQuickSightDashboard(viewName, sql, tableVisualViewName,
         sqlTable, query, sheetId);
+      if (result.dashboardEmbedUrl === '' && query.action === ExploreRequestAction.PREVIEW) {
+        console.log(result);
```
Comment (zxkane) [inline src/control-plane/backend/lambda/api/service/reporting.ts:131]: use logger

File: src/control-plane/backend/lambda/api/model/stacks.ts [hunk 5]
```diff
@@ -196,6 +196,12 @@ export class CIngestionServerStack extends JSONObject {
 
   @JSONObject.required
   @JSONObject.gt(0)
+  @JSONObject.custom( (_stack:CIngestionServerStack, _key:string, value:number) => {
+    if (value === 1) {
+      throw new ClickStreamBadRequestError('ServerMax must greater than 1.');
```
Comment (zxkane) [inline src/control-plane/backend/lambda/api/model/stacks.ts:201]: -> must be greater

**Review by chenhaiyun — APPROVED** [review]

**Review by zxkane — CHANGES_REQUESTED** [review]

**Review by zxkane — APPROVED** [review]

**Pull request merged**